ui/ember-cli-build.js

```diff
@@ -42,6 +42,8 @@ const appConfig = {
     implementation: nodeSass,
     sourceMap: false,
     onlyIncluded: true,
+    precision: 4,
```

Comment:
I suggest, if not done yet, to add also this:
```
    minifyCSS: {
      options: {
        advanced: false,
      },
    },
```
see how it's done in Cloud UI for example: https://github.com/hashicorp/cloud-ui/blob/main/hcp/ember-cli-build.js#L214-L218

Reply:
Thanks for pointing this out @didoo. I'm not seeing any mention of this option in the [broccoli-clean-css docs](https://github.com/shinnn/broccoli-clean-css/blob/master/README.md) or the [clean-css docs](https://github.com/clean-css/clean-css/blob/master/README.md). Do you have any information on what that option does and/or why it's needed? It might be a good idea to add it to the [Getting started for engineers](https://helios.hashicorp.design/getting-started/for-engineers) section of the Helios docs as well.

Reply:
> I'm not seeing any mention of this option in the [broccoli-clean-css docs](https://github.com/shinnn/broccoli-clean-css/blob/master/README.md) or the [clean-css docs](https://github.com/clean-css/clean-css/blob/master/README.md). Do you have any information on what that option does and/or why it's needed?

@zofskeez it's some arcane ember "magic" that we found was messing up the generation of CSS (essentially was super-optimizing the CSS, beyond what is normally done, by even changing the order of the CSS declarations which is a no-no in CSS, where order matters).

A couple of links for reference:
- https://github.com/hashicorp/cloud-ui/pull/3112
- https://github.com/ember-cli/ember-cli/issues/8457 
- super mega thread: https://hashicorp.slack.com/archives/CPB8GS9QT/p1658220312956889

> It might be a good idea to add it to the [Getting started for engineers](https://helios.hashicorp.design/getting-started/for-engineers) section of the Helios docs as well.

Good point, I'll open a PR immediately!

Reply:
https://github.com/hashicorp/design-system/pull/1219

Reply:
Great thanks for the additional context on this. It looks like it was a tough one to track down so I appreciate pointing it out and updating the docs!